app/services/record_declaration.rb

```diff
@@ -219,6 +219,14 @@ def validates_billable_slot_available
     errors.add(:base, I18n.t(:declaration_already_exists))
   end
 
+  def participant_declaration_class
+    if participant_profile.mentor?
```

Comment:
Should we take in consideration the course identifier sent in the params as well?
`ecf-induction` `ecf-mentor`

Reply:
I think we should be good just checking the profile type here; the `CourseValidator` uses the profile to determine if the course is valid, so it won't go through regardless